resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -497,6 +497,13 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_KERBEROS_SUPPORT =
+    ConfigBuilder("spark.kubernetes.kerberos")
+      .doc("Specify whether your job is a job " +
+        "that will require a Delegation Token to access HDFS")
```

Comment:
If the entire doc can fit on one line then make it fit on one line. Otherwise break the line as far to the right of the sentence as possible.